scripts/generate.ts

```diff
@@ -31,8 +39,13 @@ async function logProgress(message: string, body: () => Promise<void>) {
   process.stderr.write("done\n");
 }
 
-async function main({ clean }: { clean: boolean }) {
-  const repoRoot = path.resolve(__dirname, "..");
+const repoRoot = path.resolve(__dirname, "..");
+
+const pythonSdkRoot = path.resolve(repoRoot, "python", "foxglove-sdk");
+const pythonSdkSourceRoot = path.join(pythonSdkRoot, "src", "generated");
+const pythonSdkStub = path.join(pythonSdkRoot, "python", "foxglove", "_foxglove_py", "schemas.pyi");
+
+async function main({ clean, includeSdk }: { clean: boolean; includeSdk: boolean }) {
   const outDir = path.join(repoRoot, "schemas");
   const rosOutDir = path.join(repoRoot, "ros_foxglove_msgs");
   const typescriptTypesDir = path.join(repoRoot, "typescript/schemas/src/types");
```

Comment:
looks like you could hoist these up above the main function too - keep them all in one place

Reply:
`--include-sdk` is now gone, so I moved everything back into `main`, and these are all in one place again